pinot-core/src/main/java/org/apache/pinot/core/query/selection/SelectionOperatorService.java

```diff
@@ -179,4 +180,31 @@ public SelectionResults renderSelectionResultsWithOrdering(boolean preserveType)
 
     return new SelectionResults(_selectionColumns, rowsInSelectionResults);
   }
+
+  /**
+   * Render the selection rows to a {@link ResultTable} object for selection queries with
+   * <code>ORDER BY</code>. (Broker side)
+   * <p>{@link ResultTable} object will be used to build the broker response.
+   * <p>Should be called after method "reduceWithOrdering()".
+   *
+   * @return {@link SelectionResults} object results.
+   */
+  public ResultTable renderResultTableWithOrdering() {
+    LinkedList<Object[]> rowsInSelectionResults = new LinkedList<>();
```

Comment:
We can use the size of rows

Reply:
It's a LinkedList, implementation doesn't require or permit initial size

Reply:
Sorry, my bad :)